pyproject.toml

```diff
@@ -1,14 +1,24 @@
 [tool.poetry]
-name = "pyap"
-version = "0.3.1"
-description = "Pyap is an MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
-authors = ["Vladimir Goncharov <vladimarius@gmail.com>"]
-documentation = "https://github.com/vladimarius/pyap"
+name = "argyle-pyap"
```

Comment:
Maybe we could add
https://python-poetry.org/docs/pyproject/#keywords
https://python-poetry.org/docs/pyproject/#homepage
https://python-poetry.org/docs/pyproject/#repository